numpy/ma/core.py

```diff
@@ -4200,7 +4314,7 @@ def __add__(self, other):
         """
         if self._delegate_binop(other):
             return NotImplemented
-        return add(self, other)
+        return np.add(self, other)
```

Comment:
I think that for all these, one can now safely remove the overrides, since `__array_ufunc__` will take care.